mozilla-release/browser/config/cliqz.mozconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2016 Cliqz GmbH. All rights reserved.
+# Author: Max Breev <maxim@cliqz.com>
+
+. $topsrcdir/browser/config/mozconfig
+
+ac_add_options --enable-update-packaging
+ac_add_options --enable-signmar
+
+ac_add_options --with-branding=browser/branding/cliqz
+ac_add_options --enable-official-branding
+ac_add_options --with-app-name=CLIQZ
+
+export CQZ_UI_LOCALE=`echo $LANG`
```

Comment:
This file is actually a unix shell script. It means, there should be an appropriate environment set on Windows too.
But it's true, that this particular line should be probably returned to magic sript. LANG variable is overridden there.